package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "titanium",
   "main": "./lib/index",
-  "version": "3.0.5",
```

Comment:
Missed this, please revert the version change as it's handled automatically when the package is released